packages/jade/plugin/handler.js

```diff
@@ -47,6 +82,20 @@ var getCompilerResult = function (compileStep, fileMode) {
   }
 };
 
+// Generate helper functions using function name - function body map
+//   in the input.
+var templateHelperGen = function (helpers, tplName) {
```

Comment:
`templateHelperGen` and `templateEventsGen` are very similar, wouldn’t be better to use the same underlining function?